install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -445,11 +425,22 @@ metadata:
   name: slow
 provisioner: kubernetes.io/azure-disk
 parameters:
-  storageaccounttype: Standard_LRS  <1>
-  kind: Shared  <2>
+  storageAccount: azure_storage_account_name  <1>
+  storageaccounttype: Standard_LRS  <2>
+  kind: Shared  <3>
```

Comment:
Here the example of `kind` better with value of `Dedicated`. Since actually `Shared` does not work with `storageAccount` together. As the bug title, If you want to specify your storage account, the kind should be `Dedicated`